### DISCUSSION TITLE
无法复现单张图片测试样例结果

### DISCUSSION THREAD

**Discussion by shendushushi-jase** [PaddlePaddle/PaddleOCR#12422] [Q&A]
style_text_models  中的模型或者字体有过调整，无法复现测试效果；
style_text_models  模型下载地址：wget https://paddleocr.bj.bcebos.com/dygraph_v2.0/style_text/style_text_models.zip
执行测试的命令：
python3 tools/synth_image.py -c configs/config.yml --style_image examples/style_images/2.jpg --text_corpus PaddleOCR --language en
生成的fake_text.jpg 图片如下：
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/82139207/d53cda93-79ae-4f91-a957-8a6a54f41e69)

**Comment by Sencc** [comment]
我测试出来和上面一样的效果，请问是什么问题呢？

**Comment by fnwkqwan** [comment]
哈哈，我也是，这是啥情况啊？

**Comment by Sencc** [comment]
> 哈哈，我也是，这是啥情况啊？

估计是en字体有问题，换个就行了
